main.go

```diff
@@ -10,19 +10,21 @@ import (
 )
 
 const (
-	version = "0.0.1"
+	version  = "0.0.1"
+	logLevel = "debug"
```

Comment:
See previous comment. Can we make this an option for the user? Should be simple enough using the CLI library.

Reply:
The log level sure, the version is for us ;) and the use of go-releaser